src/lib.rs

```diff
@@ -474,7 +474,12 @@ impl Build {
 
     fn run_command(&self, mut command: Command, desc: &str) {
         println!("running {:?}", command);
-        let status = command.status().unwrap();
+        let status = match command.status() {
+            Ok(value) => value,
+            Err(error) => {
+                panic!("Error running command {:?} => {:?}", command, error);
```

Comment:
I don't think the command needs to be printed again because it was already printed a few lines above this?

Reply:
Done